packages/web-components/@sfx/sayt/test/unit/sayt.test.ts

```diff
@@ -222,6 +229,120 @@ describe('Sayt Component', () => {
     });
   });
 
+  describe('requestSayt()', () => {
+    let dispatchEvent, query, searchbox;
+    beforeEach(() => {
+      query = 'some-query';
```

Comment:
This default is only used in one test, so I think we can remove `query` as a shared variable and use it only in the tests.